DiscordBot/Commands/EventCommands.cs

```diff
@@ -17,6 +17,9 @@ namespace DiscordBot.Commands
     internal class EventCommands : BaseCommandModule
     {
         public const string ClearReaction = ":no_entry_sign:";
+        public const string RefreshReaction = ":arrows_counterclockwise:";
+
+        public const string SignupChannelName = "events";
```

Comment:
Do these new consts need to be public? 🙂

Reply:
RefreshReaction should have been used in Program.cs but SignupChannelName can be private.